x-pack/plugins/apm/public/components/app/correlations/ml_failure_correlations.tsx

```diff
@@ -0,0 +1,410 @@
+/*
```

Comment:
Suggest to name the file and components without `ml` prefix (`failure_correlations.tsx`). We kept the prefix around for 7.14 because the old code was still around. With the previous components planned to go for 7.15 we no longer will need the prefixes.

Reply:
Updated here https://github.com/elastic/kibana/pull/108441/commits/155443605ee491b2eb7c2bcb7b3e458eed9591e6